vector-im/element-android
# [Location sharing] - Update DB entity when a live is timed out

## Type of change


## Content


Introducing `DeactivateLiveLocationShareWorker` to set the `isActive` field to `false` after a live location share is timed out.

## Motivation and context


Closes #6123 
It enables also auto-refresh of the message tile when on timeline view when a live has just timed out.
It will ease the request on database to find all currently active lives in a room.

## Screenshots / GIFs





TODO


## Tests



- Ensure the location sharing in enabled in Settings -> Preferences
- Ensure the live location feature is enabled in Settings -> Labs 
- Go to a room
- Start sharing your live location for 15 minutes
- Take a coffee and wait for 15 minutes
- Check the timeline the message tile is refreshed from running state to stopped state 

## Tested devices

- OS version(s): Android 11

## Checklist
